napari/_qt/qt_event_loop.py

```diff
@@ -248,6 +257,18 @@ def _ipython_has_eventloop() -> bool:
         return False
 
 
+def _try_enable_ipython_gui(gui='qt'):
+    """Start %gui qt the eventloop."""
+    try:
```

Comment:
https://github.com/matplotlib/matplotlib/blob/200ebe1dcb8c8adb9ed4d177f02a57686ffef085/lib/matplotlib/pyplot.py#L124-L134

It is better to check if IPython is in `sys.modules` as trying to import it will succeed even if the user has not previously imported it but it is installed.

Reply:
sounds good.  but `get_ipython()` would be still `None` in that case right?